dvmconsole/MainWindow.xaml.cs

```diff
@@ -1117,6 +1117,7 @@ private void MainWindow_Loaded(object sender, RoutedEventArgs e)
             menuToggleLockWidgets.IsChecked = settingsManager.LockWidgets;
             menuSnapCallHistory.IsChecked = settingsManager.SnapCallHistoryToWindow;
             menuTogglePTTMode.IsChecked = settingsManager.TogglePTTMode;
+            menuStayOnTop.IsChecked = settingsManager.KeepWindowOnTop;
```

Comment:
If we're using StayOnTop as the menu item checkbox then we should be consistent with the User Profile option; either change the MainWindow menuStayOnTop and ToggleStayOnTop_Click to menuKeepWindowOnTop and ToggleKeepWindowOnTop_Click or change KeepWindowOnTop to StayOnTop.